src/Compilers/CSharp/Test/Semantic/Semantics/TopLevelStatementsTests.cs

```diff
@@ -5263,20 +5266,13 @@ public static async Task Wait()
         public void ExplicitMain_10()
         {
             var text = @"
-using System.Threading.Tasks;
-
-System.Console.Write(""Hi!"");
```

Comment:
Why the test scenario is drastically modified?

Reply:
Sure. We can rename the type name